packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -35,7 +35,7 @@
       @panel-width-updated="handlePanelWidthUpdated"
     >
       <template #panel1="{isDragging}">
-        <HideDuringScreenshotOrRunMode
```

Comment:
This is changed since we already know for sure it is open mode, the run mode scenario doesn't exist.